### DISCUSSION TITLE
is it possible to use vim-devicons instead of nvim-web-devicons?

### DISCUSSION THREAD

**Discussion by fegyi001** [nvim-tree/nvim-tree.lua#2044] [Q&A]
Hi,

I really like nvim-tree. However, I'm not fully satisfied with nvim-ree/nvim-web-devicons. I found another devicons plugin (ryanoasis/vim-devicons), but nvim-tree does not seem to recognize it. Is it possible for nvim-tree to make it use this devicons package?

thanks

**Comment by alex-courtis** [comment]
nvim-web-devicons is a fork of vim-devicons, adding colour.

What's the issue with nvim-web-devicons? What functionality is needed from nvim-tree or nvim-web-devicons?

Yes, it is possible to allow plugging in a different icon supplier, however we would need a very good reason to do so.
